airbyte-integrations/connectors/source-iterable/unit_tests/test_streams.py

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+import requests
+import responses
+from source_iterable.api import ListUsers
+
+
+@responses.activate
+def test_list_users_parse_response():
+    emails_sample = [
+        "te.st1@mail.com",
+        "test2@mail.com",
+        "11/test3@mail.com",
```

Comment:
Technically speaking this is correct email address: https://datatracker.ietf.org/doc/html/rfc3696#section-3 You better contact iterable support, maybe they have special quoting rules for such addresses.

Reply:
Well, seems that we're not granted to ask the support, going to try to research it myself.
![image](https://user-images.githubusercontent.com/14863647/144278813-501662c7-2c36-44a1-a29a-cffc4f3765f2.png)

Reply:
Posted the forum topic. Hope to get the answers there.